docs/tutorial/zh_CN/src/task_tracing.py

```diff
@@ -52,7 +52,20 @@
     # 连接到 OpenTelemetry 兼容的后端
     agentscope.init(tracing_url="https://your-tracing-backend:port/traces")
 
-以 Arize-Phoenix 和 Langfuse 为例：
+以阿里云云监控、Arize-Phoenix 和 Langfuse 为例：
+
+**阿里云云监控（Alibaba Cloud CloudMonitor）**：全托管可观测平台。
+
+.. code-block:: python
+    :caption: 连接到阿里云云监控
+
+    agentscope.init(tracing_url="https://tracing-cn-hangzhou.arms.aliyuncs.com/adapt_xxx/api/otlp/traces")
+
+.. tip::
+    **获取 Endpoint：** 在 `ARMS 控制台 <https://arms.console.aliyun.com/>`_ 的 **接入中心** > **OpenTelemetry** 中，
+    根据实际部署地域选择对应的 **公网接入点**。可通过环境变量 ``OTEL_SERVICE_NAME`` 自定义应用名称。
+    阿里云云监控可通过 `LoongSuite <https://github.com/alibaba/loongsuite-python-agent>`_ 探针提供零侵入的自动化接入。
```

Comment:
Shall we provide a more detailed link for users to install Loongsuite and send OTLP data to Alibaba cloud monitor?